pkgs/development/libraries/science/math/cudnn/7.5-5.0/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv
+, requireFile
+, cudatoolkit
+}:
+
+stdenv.mkDerivation rec {
+  version = "5.0";
+  cudatoolkit_version = "7.5";
+
+  name = "cudatoolkit-${cudatoolkit_version}-cudnn-${version}";
+
+  src = requireFile rec {
+    name = "cudnn-${cudatoolkit_version}-linux-x64-v${version}-ga.tgz";
+    message = ''
+      This nix expression requires that ${name} is
+      already part of the store. Register yourself to NVIDIA Accelerated Computing Developer Program
+      and download cuDNN library at https://developer.nvidia.com/cudnn, and store it to the nix store with nix-store --add-fixed sha256 <FILE>.
+    '';
```

Comment:
it is also possible to use `nix-prefetch-url` but I don't know whether one is preferred over the other.